Hub api migration guide (#9211)
camunda/camunda-docs: docs/apis-tools/hub-api-saas/authentication.md

## The process

> Generate a [JSON Web Token (JWT)](https://jwt.io/introduction/), and include it in every request. If you already have a Web Modeler API v1 token, you can use the same token for this API.

Reviewer: ```suggestion
Generate a [JSON Web Token (JWT)](https://jwt.io/introduction/), and include it in every request. If you already have a client or token for the Web Modeler API v1, then you can reuse it for this API.
```

Tokens are short-lived. I think it's more important for users to know that they don't need to recreate their API clients.
